### PR TITLE
docs: Commit Message Prefixes

### PR DESCRIPTION
**Pull request by aiAdrian**

### CODE REVIEW

**Comment by aiAdrian** [comment]
> This is a duplicate of https://github.com/OpenRailAssociation/netzgrafik-editor-frontend/blob/main/CONTRIBUTING.md?plain=1#L210-L219; we should choose which one is the right place

Thank you - i just tried to add as well some step-by-step guide how to 'change' commits by applying rebase ... 

If you think this has no value -> reject / close my PR ... at least i have no understood how i can fix it

**Review by emersion — COMMENTED** [review]
If we want to help contributors with Git usage, I would prefer to link to well-written articles on external websites instead of having our own special Git tutorials. I'm not sure it's required: I think we can provide customized help as needed in PR discussions.

File: documentation/technical/commit-prefixes.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+4. Adjust the commit message
+   Git will open an editor for each reword commit. Update the message to follow the prefix rules, for example:
+
+```bash
+feat: add user profile page
+fix: handle null value in parser
+refactor: simplify validation logic
+```
+
+5. Force-push the updated history
```
Comment (louisgreiner) [inline documentation/technical/commit-prefixes.md]: Indexes should be:
3. Adjust the commit message
4. Force-push the updated history

File: documentation/technical/commit-prefixes.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+test(parser): add edge case tests
+
+## Fixing commit messages (step-by-step)
+
+If a commit message does not follow the required prefix format, you can fix it using an interactive rebase.
+
+1. Start an interactive rebase
+
+```bash
+git rebase -i HEAD~<n>
```
[the remark this answers is not in the document]
Comment (aiAdrian) [reply documentation/technical/commit-prefixes.md]: What does it mean - if i have to fix the text . please help me . it would be very powerful having a "nice cookbook - step-by-step" .
